v2/README.md

```diff
@@ -3,6 +3,9 @@ title: Azure Service Operator v2
 type: docs
 description: "Manage your Azure resources from within your Kubernetes cluster."
 ---
+
+<img src="https://azure.github.io/azure-service-operator/favicons/favicon-256.png" style="float:right; "/>Azure Service Operator (ASO) allows you to deploy and maintain Azure Resources using the Kubernetes tooling you already know and use. Instead of separately deploying your [Redis Cache](https://azure.github.io/azure-service-operator/reference/cache/) or [PostgreSQL database server](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/) and then configuring your Kubernetes application to use them, you can use ASO to deploy and manage these resources from within your Kubernetes cluster. ASO will then automatically configure your application to use them.
```

Comment:
Not sure if I'm convinced about the logo there. It looks a bit odd to my eye.

Reply:
Couple alternative possibilities:
1. A slightly smaller variant directly under the Azure Service Operator v2 title?
2. A smaller variant at the start (or end) of the Azure Service Operator v2 title (as part of the title)

Reply:
I can reduce it to 128px square - but we can't preview because of the use of absolute references to the images.
I think 32px square is too small.

I _can_ put the 32x32 variant in the description, but not the title. It seems the right size for that location.